docs/debugging-reference.md

```diff
@@ -21,7 +21,7 @@ space so drop by our [Gitter channel][gitter] to help out make it better.
 ## Debugging via an editor
 
 If you want to debug your code with Bloop via *an editor* such as VS Code,
-vim, Sublime or emacs you can use [Metals][metals].
+Vim or Emacs you can use [Metals][metals].
```

Comment:
We haven't really had any info about debug happening with Sublime, so I figured it made sense to remove reference to it in the debugging section and instead focus on the clients I know have that capability.

Reply:
BTW sublime should be able to debug using this: https://github.com/daveleroy/sublime_debugger
But since CodeLens https://github.com/sublimelsp/LSP/issues/781 is still missing is LSP, I haven't tried yet.

Reply:
Ah cool! Do you know if anyone has by chance used it with Metals yet? Also, with the recent changes merged into Metals, you'll be able to trigger debugging w/o code lenses.